src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -54,13 +54,13 @@ def get_data(
         data_keys.POPULATION.ACMR: load_standard_data,
         data_keys.POPULATION.LIVE_BIRTH_RATE: load_standard_data,
         # TODO - add appropriate mappings
-        # data_keys.DIARRHEA.PREVALENCE: load_standard_data,
-        # data_keys.DIARRHEA.INCIDENCE_RATE: load_standard_data,
-        # data_keys.DIARRHEA.REMISSION_RATE: load_standard_data,
-        # data_keys.DIARRHEA.CSMR: load_standard_data,
-        # data_keys.DIARRHEA.EMR: load_standard_data,
-        # data_keys.DIARRHEA.DISABILITY_WEIGHT: load_standard_data,
-        # data_keys.DIARRHEA.RESTRICTIONS: load_metadata,
+        data_keys.ALZHEIMERS.PREVALENCE: load_standard_data,
+        data_keys.ALZHEIMERS.INCIDENCE_RATE: load_standard_data,
+        data_keys.ALZHEIMERS.REMISSION_RATE: load_alzheimers_remission_rate,
```

Comment:
I think it would be clearer to call this `make_rate_of_zero` or something.  But is it actually needed?